CodingGuidelines.md

```diff
@@ -38,29 +38,31 @@ This ensures documentation is properly generated and disseminated for all all cl
 
 ## MRTK namespace rules
 
-The vNext structure adheres to a strict namespace culture of mapping the namespace 1-1 with the folder structure of the project.  This ensures that classes are easy to discover and maintain.  It also ensures the dependencies of any class are laid out in the beginning using definitions of the file.
+The Mixed Reality Toolkit uses a feature based namespace model, where all foundational namespaces begin with "Microsoft.MixedReality.Toolkit". In general, you need not specify the toolkit layer (ex: Core, Providers, Services) in your namespaces.
 
-![](External/ReadMeImages/MRTK-NameSpaceExample.png)
+The currently defined namespaces are:
+
+- Microsoft.MixedReality.Toolkit
+- Microsoft.MixedReality.Toolkit.Boundary
+- Microsoft.MixedReality.Toolkit.Diagnostics
+- Microsoft.MixedReality.Toolkit.Editor
```

Comment:
Won't this collide with the unity namespace of the same name?

Reply:
The foundation code will utilize intelligent diambiguation. There will also be documentation / samples for how to disambiguate cleanly in customer code.

Reply:
> intelligent diambiguation

What does that mean?

Reply:
it means we will handle disambiguation in the most appropriate manner (ex: UnityEditor.Editor, etc)